Resources/Prototypes/Entities/Structures/plastic_flaps.yml

```diff
@@ -0,0 +1,65 @@
+- type: entity
+  id: PlasticFlapsClear
+  parent: BaseStructureDynamic
+  name: plastic flaps
+  suffix: Clear
+  description: Heavy duty, plastic flaps. Definitely can't get past those. No way.
+  placement:
+    mode: SnapgridCenter
+  components:
+  - type: Sprite
+    netsync: false
+    sprite: Structures/plastic_flaps.rsi
+    state: plasticflaps
+  - type: Physics
+  - type: Transform
+    anchored: true
+  - type: Fixtures
+    fixtures:
+    - shape:
+        !type:PhysShapeAabb
+        bounds: "-0.49,-0.49,0.49,0.49"
+      mass: 100
+      mask:
+      - Passable
+      layer:
+      - MobImpassable
```

Comment:
Shouldn't this just be like VaultImpassable and maybe Impassable?

Reply:
![vaultimpassaible](https://user-images.githubusercontent.com/47410468/153698019-d96d39d9-9aa7-4915-aff2-573caa66f010.gif)

I believe the only thing that passes through vaultimpassable is gasses and vapors

Reply:
I thought mobs weren't supposed to pass through flaps anyway, just items?

So just use Impassable if you also want thrown stuff going through and adjust elsewhere?

Reply:
In SS13 mobs in the prone state (laying down) can go through flaps.

Reply:
layer and mask still need fixing. Plastic one also still has the cooked mask.

Reply:
I went with a big experimental change to see what you think about it because it's pretty sweeping changes in regards to trying to adapt VaultImpassable to actual items.

The changes listed basically define VaultImpassable as "Human or vending machine sized" so that anything below it can go through.

Additionally changed standingstate to temporarily remove it to allow for gliding underneath conveyors when down.